app/mailers/invite_responsible_body_user_mailer.rb

```diff
@@ -0,0 +1,22 @@
+class InviteResponsibleBodyUserMailer < ApplicationMailer
+  def invite_user_email
+    @user = params[:user]
+    rb_name = @user.responsible_body.local_authority_official_name || @user.responsible_body.name
```

Comment:
How come we're not just using `responsible_body.name`?

any thoughts @fofr? The `local_authority_official name` corresponds to the official name in https://www.registers.service.gov.uk/registers/local-authority-eng

Reply:
When I was testing emails, the rb name was Stoke-on-Trent which sounded a bit odd in the context of the email e.g. invite additional users from Stoke-on-Trent instead of Stoke-on-Trent City Council so I went with the long name if we had one.